clustered/client/src/main/java/org/ehcache/clustered/client/internal/service/DefaultClusteringService.java

```diff
@@ -411,7 +411,7 @@ protected boolean isStarted() {
         clientStoreConfiguration, configuration.isAutoCreate());
       clusterTierEntities.put(cacheId, storeClientEntity);
     } catch (EntityNotFoundException e) {
-      throw new CachePersistenceException("Clustered tier proxy '" + cacheIdentifier.getId() + "' for entity '" + entityIdentifier + "' does not exist.", e);
+      throw new CachePersistenceException("cluster tier proxy '" + cacheIdentifier.getId() + "' for entity '" + entityIdentifier + "' does not exist.", e);
```

Comment:
Here also we should stick to clustered, because its a user exception and user configured a "clustered"  tier.